vllm/model_executor/models/phi3v.py

```diff
@@ -509,8 +510,8 @@ class Phi3VForCausalLM(nn.Module, SupportsMultiModal, SupportsPP):
 
     def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
         super().__init__()
+        SupportsQuant.__init__(self, vllm_config.quant_config)
```

Comment:
It seems we fail to capture the benefit of having a mixing if we have to manually call it, I'm not sure this is right as-is

Reply:
I've rewritten this to use the __new__ method